ui/00_message/common/stage_list.xml

```diff
@@ -1545,14 +1559,17 @@
   <message>
     <key>STAGE_NAME_402</key>
     <original>緑夜の庵</original>
+    <translation>Green Nights Retreat</translation>
   </message>
   <message>
     <key>STAGE_NAME_403</key>
     <original>ノーザナス侵林</original>
+    <translation>Northern Fores</translation>
```

Comment:
Damn I thought I changed it, but did not push the change, in any case I deleted this translation since its not accurate